lib/internal/test_runner/harness.js

```diff
@@ -217,6 +220,17 @@ async function startSubtest(subtest) {
 }
 
 function runInParentContext(Factory) {
+  function validateTestOptions(options) {
```

Comment:
Any specific reason this function should be nested? (and not declared on top level)

Reply:
It was my initial thought to declare it nested. But seeing it now, it would be better declared at the top level. I added that change to my recent commit